spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaHealthIndicator.java

```diff
@@ -97,6 +97,9 @@ else if (lastFetch > clientConfig.getRegistryFetchIntervalSeconds() * 2000) {
 		}
 		Map<String, Object> result = new HashMap<>();
 		for (Application application : applications.getRegisteredApplications()) {
+			if (application.getInstances().isEmpty()) {
+				continue;
```

Comment:
I'd prefer if not empty put rather than continue.

Reply:
You r right. thx